### PR TITLE
docs: update Dockerfile

### PR DESCRIPTION
**Pull request by HuKeping**
I think the `tree` and `vim` are not really needed? ping @jfrazelle 

Signed-off-by: Hu Keping hukeping@huawei.com

### CODE REVIEW

**Comment by jfrazelle** [comment]
so true, what even is this dockerfile being used for? LGTM

File: docs/security/trust/trust_sandbox.md [hunk 1]
```diff
@@ -101,8 +101,6 @@ So, you'll need an entry for both the servers in your local `/etc/hosts` file.
         RUN chmod +x /usr/bin/docker \
```
Comment (thaJeztah) [inline docs/security/trust/trust_sandbox.md]: I also see that the dockerfile uses https://master.dockerproject.org/ for running Docker. Perhaps we should change that to https://get.docker.com/builds/Linux/x86_64/docker-latest ? Fine to leave that for another PR if you want

@jfrazelle wdyt?

**Comment by thaJeztah** [comment]
LGTM, found another possible improvement https://github.com/docker/docker/pull/15652#discussion_r37360058, but no problem if you don't want to change that

**Comment by thaJeztah** [comment]
Oh, more weird things; https://github.com/docker/docker/pull/15652/files#diff-b3c62cb3fc0501039ee838964c1cee7bR110

```Dockerfile
RUN cp /root/notary/fixtures/root-ca.crt /usr/local/share/ca-certificates/root-ca.crt
```

I don't see explained in this document where those fixtures should come from

@diogomonica is that example correct?

(oh, and sorry for the noise, @HuKeping, it's just that I noticed these things while reading through your PR)

**Comment by HuKeping** [comment]
Never mind, I try the sample and it woks well, but I'm not sure about your opinion of

`RUN cp /root/notary/fixtures/root-ca.crt /usr/local/share/ca-certificates/root-ca.crt`

**Comment by thaJeztah** [comment]
@HuKeping I think my brain must have temporarily shutdown; Seeing the absolute path `/root/notary..` I must have thought, "but that's not there"... please ignore :blush:

Changes LGTM, but interested to hear about https://github.com/docker/docker/pull/15652#discussion_r37360058,

File: docs/security/trust/trust_sandbox.md (hunk 1 above)
Comment (SvenDowideit) [inline docs/security/trust/trust_sandbox.md]: I presume this document used `master` because this code wasn't released until 1.8.0 - so now's the time :)

File: docs/security/trust/trust_sandbox.md [hunk 2]
```diff
@@ -101,8 +101,6 @@ So, you'll need an entry for both the servers in your local `/etc/hosts` file.
...
           && apt-get update \
           && apt-get install -y \
-          tree \
```
Comment (diogomonica) [inline docs/security/trust/trust_sandbox.md]: vim is useful to tamper data and tree is super useful to list all the keys that are generated.

**Comment by diogomonica** [comment]
This is a sandbox to play with trust, having more tools is better than having less tools.

**Comment by jfrazelle** [comment]
Ok we can close ;) thanks for PR, but I think since this is in docs and not really trying to be minimal we may as well keep it

**Comment by jfrazelle** [comment]
That being said I see no problem w maybe a different PR where vim is changed to "vim-nox" because I'm picky ;) but that's just if someone is bored or wants a contribution :D
